templates/beta-release-post.adoc

```diff
@@ -151,10 +151,24 @@ Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
 
 [source,gradle]
 ----
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'io.openliberty.tools:liberty-gradle-plugin:3.6.2'
+    }
+}
```

Comment:
What about the portion that applies the Liberty Gradle plugin? That is needed also.

`apply plugin: 'liberty'`